src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java

```diff
@@ -117,6 +127,23 @@ public void initialize() {
     colorPicker.focusedProperty().addListener((observable, oldValue, newValue) -> chatUserContextMenuRoot.setAutoHide(!newValue));
   }
 
+  private void initModerator() {
+    moderatorService.isModerator().thenAccept(isModerator -> {
```

Comment:
lets unit test both cases and whether kickGameItem and co are set to correct visibility